cpp/src/arrow/ipc/metadata_internal.cc

```diff
@@ -477,7 +477,9 @@ static Status GetDictionaryEncoding(FBB& fbb, const std::shared_ptr<Field>& fiel
 
 static KeyValueOffset AppendKeyValue(FBB& fbb, const std::string& key,
                                      const std::string& value) {
-  return flatbuf::CreateKeyValue(fbb, fbb.CreateString(key), fbb.CreateString(value));
+  auto fbb_key_ = fbb.CreateString(key);
+  auto fbb_value_ = fbb.CreateString(value);
```

Comment:
Please don't use `_` suffix. We use it only for member variable (not local variable).

Reply:
@amoeba Could you check this comment?

Reply:
Whoops. Will fix.

Reply:
Changed in 4cc60d766727fecd807c45ee9df88824a5b80029.